tests/integration/data/schemas/sync.ts

```diff
@@ -0,0 +1,86 @@
+import { handler } from '../../../../src/commands/data/schemas/sync';
+import { schemaRepositoryMock } from '../../../__helpers__/schemaRepositoryMock';
+import { validSchema } from '../../../__helpers__/schemas';
+import { createTempDirectoryManager } from '../../../__helpers__/tempDirectoryManager';
+
+describe('exh data schemas sync', () => {
+  let tempDirectoryManager;
+  let repositoryMock: ReturnType<typeof schemaRepositoryMock>;
+
+  beforeEach(async () => {
+    tempDirectoryManager = await createTempDirectoryManager();
+    repositoryMock = schemaRepositoryMock();
+  });
+
+  afterEach(async () => {
+    jest.clearAllMocks();
+    await tempDirectoryManager.removeDirectory();
+  });
+
+  it('Creates a schema', async () => {
+    const path = await tempDirectoryManager.createTempJsonFile(validSchema);
+
+    await handler({
+      sdk: undefined,
+      file: path,
+      dir: undefined,
+      dry: false,
+      ignoreVerificationErrors: false,
+    });
+
+    // Schema creation is called
+    expect(repositoryMock.createSchemaSpy).toBeCalledTimes(1);
+
+    // Property creation is called for every property
+    expect(repositoryMock.createPropertySpy).toBeCalledTimes(Object.keys(validSchema.properties).length);
+    Object.keys(validSchema.properties).forEach((key: string) => {
```

Comment:
Rather than check it has anything, should we not check the configuration? 🤔 The same strength we check the transitions with?

        undefined,
        expect.any(String),
        {
          name: key,
          configuration: validSchema.properties[key],
        }